flow/designs/asap7/jpeg_lvt/rules-base.json

```diff
@@ -52,7 +52,7 @@
         "compare": "<="
     },
     "detailedroute__route__drc_errors": {
-        "value": 0,
+        "value": 1,
```

Comment:
Do you understand the cause of this drv?

Reply:
No I did not. But the last time we had a situation like this, I was told that this sort of small instability is expected and Osama is working on some new rip up and reroute logic which might mitigate this.